tools/probes/probes_main.c

```diff
@@ -209,17 +209,6 @@ int process_sync(struct probe_data_packet **packet, uint8_t **w_ptr, uint32_t *t
 	return 0;
 }
 
-static bool sync_word_at(uint8_t *buf, size_t len)
-{
-	if (len < sizeof(uint32_t))
```

Comment:
Can you say in the commit message why the condition can be simplified (i.e. why do we not need length anymore)

Reply:
Sure. Updated the commit message.